### PR TITLE
164901161-task: implement configuration of editorconfig

### PR DESCRIPTION
**Pull request by BukkyOmo**
## Description
 Configuration of editorconfig

## Type of change
(task)none breaking change.

## How Has This Been Tested?
This has not been tested yet

## Checklist:
 - Integrate .editorconfig file to the root folder.
- install editorconfig package

## PIVOTAL BOARD
[#164901161](https://www.pivotaltracker.com/n/projects/2320805/stories/164901161)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+USERNAME=postgres
```
Comment (paradisekelechi) [inline .env]: We do not need to make changes to this file

File: index.js [hunk 2]
```diff
@@ -1,60 +1,73 @@
-const fs = require("fs"),
```
Comment (paradisekelechi) [inline index.js]: You should not touch this file

**Review by paradisekelechi — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (BukkyOmo) [reply .env]: Okay

File: index.js (hunk 2 above)
Comment (BukkyOmo) [reply index.js]: I had to download the editorconfig npm and import it into this file

**Review by paradisekelechi — APPROVED** [review]

**Pull request merged**